### PR TITLE
Update create.scatterplot documentation

### PR DESCRIPTION
**Pull request by forbiddenpersimmon**
## Description
Updated `create.scatterplot` `type` parameter documentation to be more descriptive from just "Plot type". Includes commonly used elements for this parameter and points to `xyplot` documentation for more information.

-   [X] This PR does NOT contain [PHI](https://ohrpp.research.ucla.edu/hipaa/) or germline genetic data. A repo may need to be deleted if such data is uploaded. Disclosing PHI is a [major problem](https://healthitsecurity.com/news/ucla-health-reaches-7.5m-settlement-over-2015-breach-of-4.5m).

-   [X] This PR does NOT contain molecular files, compressed files, output files such as images (*e.g.* `.png`, .`jpeg`), `.pdf`, `.RData`, `.xlsx`, `.doc`, `.ppt`, or other non-plain-text files. To automatically exclude such files using a [.gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files) file, see [here](https://github.com/uclahs-cds/template-base/blob/main/.gitignore) for example.

-   [X] I have read the [code review guidelines](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3187646/Code+Review+Guidelines) and the [code review best practice on GitHub check-list](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3189956/Code+Review+Best+Practice+on+GitHub+-+Check+List).

-   [X] I have set up or verified the `main` branch protection rule following the [github standards](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3190380/GitHub+Standards#GitHubStandards-Branchprotectionrule) before opening this pull request.

-   [X] The name of the branch is meaningful and well formatted following the [standards](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3189956/Code+Review+Best+Practice+on+GitHub+-+Check+List), using [AD_username (or 5 letters of AD if AD is too long)]-[brief_description_of_branch].

-   [ ] I have added the major changes included in this pull request to the `NEWS` under the next release version or unreleased, and updated the date. I have also updated the version number in `DESCRIPTION` according to [semantic versioning](https://semver.org/) rules.

-   [x] Both `R CMD build` and `R CMD check` run successfully.

### CODE REVIEW

File: man/create.scatterplot.Rd [hunk 1]
```diff
@@ -203,7 +203,7 @@ create.scatterplot(
     \item{ygrid.at}{Vector listing where the y-axis grid lines should be drawn, defaults to yat}
     \item{grid.colour}{ability to set individual grid line colours}
     \item{horizontal}{xyplot-specific function that allows you to change if type='h' draws lines to the vertical or horizontal axis}
-    \item{type}{Plot type}
+    \item{type}{Accepts character vector of one or more elements defining how x and y are to be plotted. Accepted elements include: "p", "l", "h", "s", "g", "r". For other options see "type" parameter in "xyplot" documentation.}
```
Comment (aholmes) [inline man/create.scatterplot.Rd:206]: Consider describing what these characters represent.

File: man/create.scatterplot.Rd [hunk 2]
```diff
@@ -900,6 +900,7 @@ create.scatterplot(
     yaxis.fontface = 1,
     xlab.cex = 1.5,
     ylab.cex = 1.5,
+    # To plot ROC curve, add "s" or "S" to type vector
```
Comment (aholmes) [inline man/create.scatterplot.Rd:903]: The `type` description specifies `s` but not `S`. If both are supported, mention that the characters are case-insensitive.

File: man/create.scatterplot.Rd (hunk 1 above)
Comment (forbiddenpersimmon) [reply man/create.scatterplot.Rd:206]: Fine with adding more to describe, but would it be too long of a description if I do?

**Review by dan-knight — DISMISSED** [review]
Love this. I've always wondered, are these values directly from lattice?

**Comment by jarbet** [comment]
This is very helpful, I always wondered what all the accepted values were.  Thanks @forbiddenpersimmon !

**Comment by forbiddenpersimmon** [comment]
> Love this. I've always wondered, are these values directly from lattice?

Yeah these come directly from lattice's xyplot()

**Review by dan-knight — APPROVED** [review]

**Pull request merged**